packages/api-kit/tests/e2e/confirmTransaction.test.ts

```diff
@@ -32,8 +36,9 @@ describe('proposeTransaction', () => {
   it('should allow to create and confirm transactions signature using a Safe signer', async () => {
     const safeTransactionData: SafeTransactionDataPartial = {
       to: safeAddress,
-      value: '100000000000000000', // 0.01 ETH
-      data: '0x'
+      value: '10000000000000000', // 0.01 ETH
```

Comment:
Is the comment on the right-hand side still correct? You could use the viem#parseEther and remove the comment altogether.